doc/TOR.md

```diff
@@ -31,6 +31,91 @@ To provide the node with a .onion address you can:
 
 * create a **persistent** address with a hidden service.
 
+### Quick Start On Linux
+
+It is easy to create a single persistent Tor address and not announce a public IP.
+This is ideal for most setups where you have an ISP-provided router connecting your
+Internet to your local network and computer, as it does not require a stable
+public IP from your ISP (which might not give one to you for free), nor port
+forwarding (which can be hard to set up for random cheap router models).
+Tor provides NAT-punching for free, so even if you or your ISP has a complex
+network between you and the Internet, as long as you can use Tor you can
+be connected to.
+
+On most Linux distributions, making a standard installation of `tor` will
+automatically set it up to have a SOCKS5 proxy at port 9050.
+As well, you have to set up the Tor Control Port.
+On most Linux distributions there will be commented-out settings below in the
+`/etc/tor/torrc`:
+
+    ControlPort 9051
+    CookieAuthentication 1
+    CookieAuthFileGroupReadable 1
```

Comment:
I prefer wrapping in [multiline code-markup](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code), since it is more explicit and allows annotating with the highlight language.